python/tvm/relay/frontend/tensorflow.py

```diff
@@ -989,60 +1011,65 @@ def _impl(inputs, attr, params):
     'ConcatV2'                          : _concatV2(),
     'Conv2D'                            : _conv('conv'),
     'DecodeJpeg'                        : _decode_image(),
+    'DepthwiseConv2dNative'             : _conv('depthwise'),
```

Comment:
Couldn't understand this rearrangement !!!

Reply:
emm, just made the op list in alphabetical order.  Grouping similar op (like add, multiply) together is good, but it is not easy to figure out the group for all op considering we will add more and more ops, for example 'DecodeJpeg'. So I think alphabetical ordering here is simple and user/developer friendly.